utils/commons/filehandler.py

```diff
@@ -9,28 +9,35 @@
 class AbstractFileHandler(Protocol):
     @staticmethod
     @abstractmethod
-    def read(dir: Path, file_name: str) -> Content:
+    def read(directory: Path, file_name: str) -> Content:
         ...
 
     @staticmethod
     @abstractmethod
-    def write(dir: Path, file_name: str, content: Content) -> None:
+    def write(directory: Path, file_name: str, content: Content) -> None:
         ...
 
 
 class FileHandler:
     @staticmethod
-    def read(dir: Path, file_name: str) -> Content:
-        with open(dir / file_name, encoding="utf-8") as f:
+    def read(directory: Path, file_name: str) -> Content:
```

Comment:
Völlig in Ordnung, wobei `dir` schon eine stark konventionalisierte Abkürzung für directory ist.